tests/bearlib/aspects/BaseTest.py

```diff
@@ -15,5 +15,7 @@ def test_class(self):
         assert not isinstance(aspectbase, aspectclass)
 
     def test_init_needs_aspectclass(self):
-        with pytest.raises(AttributeError):
+        error_str = 'object has no attribute'
+        with pytest.raises(AttributeError) as exc:
```

Comment:
There was a period where lots of pytest-only tests where created in aspects.
We had already decided to use unittest compatible tests, but were still trying to understand and fix the pytest-only test problems.
This was a case of left-hand not talking with right hand.
The result is we are still approving pytest-only tests in aspects code, but trying to prevent the problem growing elsewhere.